apps/preview/next/pages/showcases/NewsletterBox/NewsletterBox.tsx

```diff
@@ -0,0 +1,99 @@
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { useState } from 'react';
+import { SfButton, SfInput, SfLink, SfIconCheckCircle, SfIconClose } from '@storefront-ui/react';
+
+export default function NewsletterBox() {
+  const [inputValue, setInputValue] = useState('');
+  const [positiveAlert, setPositiveAlert] = useState(false);
+  const [errorAlert, setErrorAlert] = useState(false);
+  const [emailDataBase, setEmailDataBase] = useState<string[]>([]);
+
+  const checkEmailDataBase = (email: string) => emailDataBase.find((element) => element === email);
+
+  const subscribeNewsletter = (event: React.FormEvent<HTMLFormElement>, email: string) => {
+    event.preventDefault();
+    if (!email) return;
+    if (checkEmailDataBase(email)) {
+      setErrorAlert(true);
+      setTimeout(() => setErrorAlert(false), 5000);
+    } else {
+      setPositiveAlert(true);
+      setEmailDataBase(() => [...emailDataBase, email]);
+      setTimeout(() => setPositiveAlert(false), 5000);
+    }
+    console.log(email);
```

Comment:
leftover

Reply:
there is a point in a task that email should appear in console but I can remove it form the code :)

Reply:
then we can keep it :)